.vitepress/config/shared.ts

```diff
@@ -75,7 +76,19 @@ export const sharedConfig = defineConfig({
       copyright: "Copyright © 2023-present Boshen & Oxc Contributors",
     },
   },
+  markdown: {
+    config(md) {
+      md.use(groupIconMdPlugin);
+    },
+  },
   vite: {
+    plugins: [
+      groupIconVitePlugin({
+        customIcon: {
+          ".oxlintrc": localIconLoader(import.meta.url, "../oxc-logo.svg"),
```

Comment:
Can we use the cdn'ed url https://cdn.jsdelivr.net/gh/oxc-project/oxc-assets/round.svg

Reply:
Current not. I will add this feature recently  to make it support load svg from a url.